model/resource/service.yaml

```
@@ -1,8 +1,10 @@
 groups:
   - id: service
     type: resource
+    name: 'service'
```

super-nit: can we use/not use quotes consistently ?

```suggestion
    name: service
```